.github/workflows/export-workspaces-as-dynamic-caller.yaml

```diff
@@ -1,4 +1,4 @@
-name: Export Dynamic Plugin Packages (Caller)
+name: Export Workspace as Dynamic Plugins Packages (Caller)
```

Comment:
Why not simply rename this workflow: `export-workspaces-as-dynamic` (the fact that it is calling a reusable workflow with the same name is an implementation detail after all) ?